docs/source/format/Flight.rst

```diff
@@ -20,7 +20,7 @@
 Arrow Flight RPC
 ================
 
-Arrow Flight is a RPC framework for high-performance data services
+Arrow Flight is an RPC framework for high-performance data services
```

Comment:
Recalled mid school English quiz :)
Looks it should be "an", https://www.quora.com/Which-is-correct-in-English-an-R-or-a-R